frontend/src/employee-frontend/components/unit/tab-unit-information/UnitAccessControl.tsx

```diff
@@ -323,7 +323,7 @@ const DevicesTable = React.memo(function DevicesTable({
         </Tr>
       </Thead>
       <Tbody>
-        {rows.map((row) => (
+        {sortBy(rows, ['name']).map((row) => (
```

Comment:
Consider using the function-based variant of `sortBy`, because lodash typings lose typing information in the "field name as string"-variants. For example:

This compiles but doesn't work (AFAIK there's no runtime error, but it just doesn't sort correctly):

```typescript
sortBy(rows, ['lolasdf'])
```

These both give a proper compilation error:

```typescript
sortBy(rows, ({ lolasdf }) => lolasdf)
sortBy(rows, (row) => row.lolasdf)
```

It's a bit longer but better for code maintainability. We still use the "bad variant" in many places, but in new code the function-based variant is better.